src/overlays/gamestates/ovl_opening/z_opening.c

```diff
@@ -1,18 +1,64 @@
 /*
  * File: z_opening.c
  * Overlay: ovl_opening
- * Description:
+ * Description: Initializes the game into the title screen
  */
 
 #include "z_opening.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_opening/Opening_SetupForTitleCutscene.s")
+void Opening_SetupForTitleCutscene(OpeningContext* this) {
+    static s32 openingEntrances[] = { 0x00001C00, 0x00001C10 };
+    static s32 openingCutscenes[] = { 0x0000FFFA, 0x0000FFFA };
```

Comment:
Are the leading zeroes necessary?


```suggestion
    static s32 openingEntrances[] = { 0x1C00, 0x1C10 };
    static s32 openingCutscenes[] = { 0xFFFA, 0xFFFA };
```

Reply:
I suppose, although it does disguise their `s32`ness.